test/e2e/e2e-test/docker/go/go2sky-e2e/go.sum

```diff
@@ -0,0 +1,79 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
Do we really need this file? I think you have the go2sky commitid, should have these.

Reply:
> Do we really need this file?

`go.sum` should be kept.

https://golang.org/cmd/go/#hdr-Module_authentication_using_go_sum

> I think you have the go2sky commitid, should have these already, if we run go vendor or something in the CI process.

I depend on `go2sky` with the [pseudo-version](https://golang.org/cmd/go/#hdr-Pseudo_versions), which is actually the same as the `commit-id` effect.

Reply:
My point is, you have the go2sky commit I'd already, you should be able to regenerate all these. Otherwise, how users use our repo?

Reply:
I know this file is needed, but not hosted

Reply:
Users can use it by adding it to [go.mod](https://github.com/arugal/skywalking/blob/5470877696289f77b40d1debb6f454417b003d27/test/e2e/e2e-test/docker/go/go2sky-e2e/go.mod#L5).

Reply:
I think you haven't followed my point. Let's talk in this way, one day, user came to the go2sky repo, he didn't need anything rather than standard Go env. Right? He could do git clone, And more commands to make things ready. Now, we have commit ID file, we should not maintain this file in main repo.
Is this clear now?